lib/msal-common/src/request/BaseAuthRequest.ts

```diff
@@ -19,6 +19,7 @@ import { AuthenticationScheme } from "../utils/Constants";
 export type BaseAuthRequest = {
     authority: string;
     correlationId: string;
+    nonce?: string;
```

Comment:
We should also remove this from `CommonAuthorizationUrlRequest` if we're moving it up a level

Reply:
Ah, I think this is left over from when I thought we needed to pass a nonce for the server-side request for Node, may not be needed anymore.

Reply:
Node ideally should support nonce, we chose not to do it in the first implementation. Keeping it here can help with adding client nonce support in node I guess.

Reply:
I'll remove for now, can add it later if needed.